### DISCUSSION TITLE
How to get slide size?

### DISCUSSION THREAD

**Discussion by mickwadev** [ShapeCrawler/ShapeCrawler#210] [Q&A] [answered]
Hello,

IPresentation SlideWidth and SlideHeight are giving me big numbers: 9144000 and 5143500
Should I scale them somehow? Are this pixels values? Or maybe real slide size is somewhere else?

Best regards,
Michał

**Comment by AdamShakhabov** [comment, marked as answer]
Hello Michal.

It is EMUs (English Metric Unit).

We've already changed this. Now, sizes will be measured in pixels. These changes will be available tomorrow with the next v0.27.0 version.

**Reply by mickwadev** [reply]
Thank You for help, devide by 9525 worked

**Comment by AdamShakhabov** [comment]
Keep it in mind that it depend on [screen resolution](http://polymathprogrammer.com/2009/10/22/english-metric-units-and-open-xml/).

Check the last [**v0.27.0**](https://github.com/ShapeCrawler/ShapeCrawler/releases/tag/v0.27.0) version. This returns pixels.

Let me know, if doesn't work as expected. We will fix that :)
